.idea/runConfigurations/flutter_idea__runIde_.xml

```diff
@@ -13,7 +13,7 @@
           <option value="runIde" />
         </list>
       </option>
-      <option name="vmOptions" />
```

Comment:
did you mean to include this change in the CL?

Reply:
No, I missed that. Although I don't think it matters.

Reply:
Oops, I was wrong. I did add `vmOptions` so removing this empty one is correct.